src/net_processing.cpp

```diff
@@ -248,6 +255,7 @@ class PeerManagerImpl final : public PeerManager
     void CheckForStaleTipAndEvictPeers() override;
     bool GetNodeStateStats(NodeId nodeid, CNodeStateStats& stats) override;
     bool IgnoresIncomingTxs() override { return m_ignore_incoming_txs; }
+    void SendPings() override;
```

Comment:
Only mentioning since you've got `RunInactivityChecks` as a separate commit, but I think you could reasonably have pulled `SendPings()` out as a separate commit prior to moving things from net to net_processing too.

Reply:
Yes, good point.